packages/iconset/stories/icons-demo.ts

```diff
@@ -182,11 +234,29 @@ export class IconsDemo extends SpectrumElement {
         this.updateSearch(event);
     }
     private renderSearch(): TemplateResult {
-        const matchingIcons = this.search
+        let matchingIcons = this.search
             ? this.icons.filter(
                   (icon) => icon.name.toLowerCase().search(this.search) !== -1
               )
             : this.icons;
+        if (this.spectrumVersion === 2) {
+            // don't show icons from s1IconsList
+            matchingIcons = matchingIcons.filter((icon) => {
+                let iconName = icon.name;
+                iconName = iconName.replace(/\s/g, '');
+                iconName = iconName.toLowerCase();
+                return iconsList['s2'].includes(iconName);
```

Comment:
Instead of repeating this logic in if and else let's write it this way!
```js
const filterIcons = (icon, spectrumType) => {
    let iconName = icon.name;
    iconName = iconName.replace(/\s/g, '');
    iconName = iconName.toLowerCase();
    return iconsList[spectrumType].includes(iconName);
}
```
// call like filterIcons(icon, 's1')